docs/blog/_posts/2018-04-26-eighth-dotty-milestone-release.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.7.0 and 0.8.0-RC1
+author: Allan Renucci
+authorImg: /images/allan.jpg
+date: 2018-04-26
+---
+
+Today, we are excited to release Dotty versions 0.7.0 and 0.8.0-RC1. These releases
+serve as a technology preview that demonstrates new language features and the compiler supporting them.
+
+If you’re not familiar with Dotty, it's a platform to try out new language concepts and compiler
+technologies for Scala. The focus is mainly on simplification. We remove extraneous syntax
+(e.g. no XML literals), and try to boil down Scala’s types into a smaller set of more fundamental
+constructs. The theory behind these constructs is researched in
+[DOT](https://infoscience.epfl.ch/record/215280), a calculus for dependent object types.
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our eighth scheduled release according to our [6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+The [previous technology preview](https://github.com/lampepfl/dotty/releases/tag/0.7.0-RC1) simplified
+enums, introduced erased terms, improved IDE support and improved pattern matching for GADT.
+
+## What’s new in the 0.8.0-RC1 technology preview?
+
+### sbt 1 support [#3872](https://github.com/lampepfl/dotty/pull/3872)
+Starting with Dotty 0.8.0, we will only support versions of sbt >= 1.1.4. Migrating to sbt 1
+lets us use the new improved incremental compiler for Scala called [Zinc](https://github.com/sbt/zinc),
+and enables integration with tools such as [Bloop](https://scalacenter.github.io/bloop/).
+
+### Unchecked warnings [#4045](https://github.com/lampepfl/dotty/pull/4045)
+Dotty now emits `unchecked` warnings like `scalac` whenever a type test is performed but cannot
+safely be checked at runtime. For example:
+
+```scala
+scala> def foo(x: Any) = x.isInstanceOf[List[String]]
+1 |def foo(x: Any) = x.isInstanceOf[List[String]]
+  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  |               the type test for List[String] cannot be checked at runtime
+```
+
+### Kind Polymorphism [#4108](https://github.com/lampepfl/dotty/pull/4108)
```

Comment:
@allanrenucci Right now this is under a compiler flag `-Ykind-polymorphism` and it's in a sort-of limbo status (some pieces needed to use it much are missing), I'm not sure how much we want to advertise it yet?